sites/partners/package.json

```diff
@@ -8,13 +8,15 @@
   "scripts": {
     "dev": "NODE_OPTIONS='--inspect=9231' next -p ${NEXTJS_PORT:-3001}",
     "build": "next build",
-    "test": "concurrently \"yarn dev\" \"cypress open\"",
-    "test:headless": "concurrently \"yarn dev\" \"cypress run\"",
+    "cypress:open": "wait-on \"http-get://localhost:${PORT:-3100}/docs\" \"http-get://localhost:${PORT:-3001}\" && cypress open",
+    "cypress:run": "wait-on \"http-get://localhost:${PORT:-3100}/docs\" \"http-get://localhost:${PORT:-3001}\" && cypress run",
+    "test": "concurrently \"yarn dev:all\" \"yarn cypress:open\"",
+    "test:headless": "concurrently \"yarn dev:all\" \"yarn cypress:run\"",
```

Comment:
Are you going to add this test to Travis?

Reply:
Added!